specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -6996,6 +6996,16 @@
       },
       "description": "Specifies the boot diagnostic settings state. <br><br>Minimum api-version: 2015-06-15."
     },
+    "BillingProfile": {
```

Comment:
adding new property to stable version is better to be in a new service tag

Reply:
@yungezz CRP has implemented PUT as a Patch. This is an optional property, so not including the property will leave it the same as before.

Reply:
Please reopen if you have any further questions. Resolving conversation.

Reply:
Do you have any further questions @yungezz ?